.github/workflows/create-release-build.yaml

```diff
@@ -0,0 +1,72 @@
+# This starter workflow is for a CMake project running on a single platform.
+name: Create Release Build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+  # Manual trigger using the Actions page.
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
```

Comment:
This var is only used in 1 place and since this is a release build workflow I think we will always want Release. So lets just inline it where it is used?

Reply:
Done.